forage-android/src/main/java/com/joinforage/forage/android/model/Card.kt

```diff
@@ -1,17 +1,13 @@
 package com.joinforage.forage.android.model
 
-import kotlinx.parcelize.Parcelize
-
 /**
  * @property last4 The last 4 digits of the card number
  */
-@Parcelize
-sealed class Card(open val last4: String, internal open val token: String) : ForageModel {
+sealed class Card(open val last4: String, internal open val token: String) {
     /**
      * @property last4 The last 4 digits of the EBT card number
```

Comment:
Super nit: The previous descriptions of `Balance` params include periods, but these property descriptions do not. Could we pick one for consistency? It looks like the Android docs tend to use periods ([example](https://android.joinforage.app/forage-android/com.joinforage.forage.android/-check-balance-params/)).

Reply:
I also think we want to capitalize EBT card -> EBT Card, even though we're not referring to the card as a standalone entity.

Reply:
Great catches! Addressed here https://github.com/teamforage/forage-android-sdk/pull/249/commits/20466f363a30f85a3a9ef246c720b1ae11f1ce4e